x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -106,6 +108,14 @@ export async function indexHostsAndAlerts(
 
     mergeAndAppendArrays(response, indexedHosts);
 
+    await indexEndpointMetricsDocs({
```

Comment:
Once you add a return type, store it here and return it back as part of `IndexedHostsAndAlertsResponse`